### PR TITLE
[Sage-290] Empty State - Foundations Update

### PR DESCRIPTION
**Pull request by anechol**
## Description

Updates Empty State component:
- Updates description text
- Adds custom content `div` for margin top spacing
- Updates image height and width for both page and compact versions
- Adjusts styling for more flexible images on all screen sizes
- Updates text `font-color` to `Grey 700`
- Fixes `compact` variant padding
- Adds description for `compact` variant
- Adds custom slot `:sage_empty_state_video` for Rails
- Adds prop `video` for React

### Implementation Updates
N/A

### Follow-Up Items
N/A

## Screenshots

|  Before  |  After  |
|--------|--------|
Screen Shot 2022-03-21 at 2 32 01 PM|Screen Shot 2022-03-21 at 2 18 34 PM
Screen Shot 2022-03-21 at 2 32 20 PM|Screen Shot 2022-03-21 at 2 18 45 PM

### Rails
- Navigate to [Empty State](http://localhost:4000/pages/component/empty_state) component
- Check that it matches Figma specs
- Check that new `video` slot works as expected

### React
- Navigate to [Empty State](http://localhost:4100/?path=/docs/sage-emptystate--default) component
- Check that it matches Figma specs
- Check that new `video` prop works as expected

## Testing in `kajabi-products`

(LOW) Updates Empty State component to Foundations design specs. Adds new video slot/prop. No effect on Kajabi Products.

## Related

[Sage-290](https://kajabi.atlassian.net/browse/SAGE-290)

### CODE REVIEW

File: docs/app/helpers/components_helper.rb [hunk 1]
```diff
@@ -544,11 +544,11 @@ def sage_components
       },
       {
         title: "empty_state",
-        description: "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.",
-        scss: "todo",
+        description: "The Empty State is displayed for main application features that have never been interacted with before. The Empty State is also used for smaller features in the app that primarily focus on data entry and have no data to show.",
```
Comment (monicawheeler) [inline docs/app/helpers/components_helper.rb:547]: 🙌

File: docs/app/views/examples/components/empty_state/_preview.html.erb [hunk 2]
```diff
@@ -49,6 +50,37 @@
...
+      Elit arcu volutpat cursus ultricies ac, ultricies.
+      Platea sed nibh molestie ut.
+    </p>
+  <% end %>
+  <% content_for :sage_empty_state_actions do %>
+    <%= sage_component SageButtonGroup, { gap: :md } do %>
+      <%= sage_component SageButton, {
+        attributes: { href: "#" },
+        style: "primary",
+        value: "Create Email Campaings",
```
Comment (monicawheeler) [inline docs/app/views/examples/components/empty_state/_preview.html.erb:78]:
```suggestion
        value: "Create Email Campaigns",
```

**Review by pixelflips — COMMENTED** [review]
@anechol with the upcoming updates to the images, how do we feel about making the image sizing a bit more flexible? Check out the design and let me know what you think.

https://www.figma.com/file/TpQHCZVS4luaIE1oYOCXQw/Illustrations?node-id=47%3A537

Could possibly be handled in a follow-up, but just thought I'd mention it here.

**Comment by anechol** [comment]
> @anechol with the upcoming updates to the images, how do we feel about making the image sizing a bit more flexible? Check out the design and let me know what you think.
> 
> https://www.figma.com/file/TpQHCZVS4luaIE1oYOCXQw/Illustrations?node-id=47%3A537
> 
> Could possibly be handled in a follow-up, but just thought I'd mention it here.

@pixelflips I totally agree with you. Responsive/flexible images is something I may want the group's input on as we've had feedback about it before, and something we may solve with either a new component or a utility class. For now, I can make these images more flexible and request another review from you when I've worked out a solution.

**Comment by pixelflips** [comment]
> @pixelflips I totally agree with you. Responsive/flexible images is something I may want the group's input on as we've had feedback about it before, and something we may solve with either a new component or a utility class. For now, I can make these images more flexible and request another review from you when I've worked out a solution.

@anechol sounds like a plan, let me know if I can help in any way. Happy to join any conversations that come up as well.

**Review by pixelflips — APPROVED** [review]
LGTM! 👍🏼

**Pull request merged**